docs/workflows/copy.rst

```diff
@@ -1,43 +1,189 @@
-                    "count": 1,
-                    "href": "/pulp/api/v3/content/rpm/advisories/?repository_version_removed=/pulp/api/v3/repositories/rpm/rpm/c8c632eb-cf9a-494f-8e12-b3e651b6e75e/versions/3/"
-                }
-            }
-        },
-        "number": 3,
-        "pulp_created": "2019-11-27T16:07:21.561748Z",
-        "pulp_href": "/pulp/api/v3/repositories/rpm/rpm/c8c632eb-cf9a-494f-8e12-b3e651b6e75e/versions/3/"
-    }
+If you want to copy RPM content from one repository into another repository, you have two options
+for doing so.
 
+
+.. _basic-modify-workflow:
+
+Basic Repository Modification API
+---------------------------------
+
+Like all Pulp repositories, you can use the ${repo_href}/modify/ endpoint to:
+
+* add or remove individual content units from a repository by HREF
+* roll back the content present in a repository to that of a previous version using 'base_version'
+* clone a repository version from another repo using 'base_version'
+
+.. code-block:: bash
+
+    export CONTENT_HHREF=/pulp/api/v3/content/rpm/packages/9ee09de7-5fff-4805-8b30-9ab95493317d/
+
+    # Add content units to the repository
+    http POST $BASE_ADDR$REPO_HREF'modify'/ add_content_units:=["$CONTENT_HREF"]
+
+    # Remove content units from the repository
+    http POST $BASE_ADDR$REPO_HREF'modify'/ remove_content_units:=["$CONTENT_HREF"]
+
+    # Clone a repository (can be composed with addition or removal of units)
+    export TASK_URL=$(http POST $BASE_ADDR$REPO_HREF'modify'/ \
+        base_version=$REPOVERSION_HREF_WITH_PKG | jq -r '.task')
+
+    # Poll the task (here we use a function defined in docs/_scripts/base.sh)
+    wait_until_task_finished $BASE_ADDR$TASK_URL
+
+    # After the task is complete, it gives us a new repository version
+    echo "Set REPOVERSION_HREF from finished task."
+    export REPOVERSION_HREF=$(http $BASE_ADDR$TASK_URL| jq -r '.created_resources | first')
+
+    echo "Inspecting RepositoryVersion."
+    http $BASE_ADDR$REPOVERSION_HREF
+
+
+.. copy-workflow:
+
+Advanced copy workflow
+----------------------
+
+.. note::
+
+    The RPM copy API is a **tech preview**, while we hope it can remain stable, it may be subject
+    to change in future releases.
+
+RPM repositories have a large number of unique use cases for which the standard 'generic' Pulp
+repository modification API is insufficient, so a separate RPM-specific API is provided for more
+'advanced' use cases.
+
+Several types of RPM content, such as Advisories (Errata), Package Groups, and Modules
+depend on the existence of other content units to be "correct" or meaningful. For example:
+
+#. An Advisory (Errata) references RPM Packages and Modules that are needed to address a
+   particular bug or security concern. In order for the Advisory to be useful, these RPM packages
+   or Modules must be present in the same repository - otherwise the Advisory is effectively "broken"
+   and won't be possible to install correctly on a client system.
```

Comment:
I'd say that otherwise the Advisory won't be fully installed on a client system.
It's possible to install it, dnf will install everything available, so only available subset will be installed.